docs/02-providers/aws/examples/hello-world/python/README.md

```diff
@@ -7,4 +7,24 @@ layout: Doc
 
 # Hello World in Python
 
-[See installation guide](/docs/01-guide/01-installing-serverless.md)
+Make sure `serverless` is installed. [See installation guide](/docs/01-guide/01-installing-serverless.md)
+
+## 1. Deploy
+
+`serverless deploy` or `sls deploy`. `sls` is shorthand for the serverless CLI command
+
+## 2. Invoke deployed function
+
+`serverless invoke --function helloWorld` or `serverless invoke -f helloWorld`
+
+`-f` is shorthand for `--function`
+
+In your terminal window you should be the response from AWS Lambda
+
+```bash
+{
+  "message": "Hello World"
+}
+```
+
+Congrats you have just deployed and ran your hello world function!
```

Comment:
I mean to put the instructions in the top level, with examples in Node/Python sections, since this is the only part that is different